site/src/pages/components/popover.research.explainer.mdx

```diff
@@ -137,7 +137,7 @@ So this markup represents popover content:
 <div popover="auto">I am a popover</div>
 ```
 
-As written above, the `<div>` will be rendered `display:none` by the UA stylesheet, meaning it will not be shown when the page is loaded. To show the popover, one of several methods can be used: [declarative triggering](#declarative-triggers), [Javascript triggering](#javascript-trigger), or [page load triggering](#page-load-trigger).
```

Comment:
I believe that "page load triggering" is referring to the defaultopen attribute which we didn't end up shipping, so I guess its ok to remove this